Fix: remove l2db

Shailu-s opened the pull request:
This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4
https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4

I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.

Shailu-s committed:
removed L2DB

Shailu-s committed:
Develop merged into it

Shailu-s committed:
removed purger and l2tx related data

Intizar-T commented:
> This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4 https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4
> 
> I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.

Agreed! It'd be great to break this PR down into multiple PRs. Separate PRs for L2Db, purger, l2tx, etc. would make it easier for reviewing and rolling back later on if something goes wrong 🙏

Shailu-s commented:
> > This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4 https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4
> > I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.
> 
> Agreed! It'd be great to break this PR down into multiple PRs. Separate PRs for L2Db, purger, l2tx, etc. would make it easier for reviewing and rolling back later on if something goes wrong 🙏
It will be time taking task to undo changes from this PR and create a new one for functions which I have already removed. I will create separate PRs for L2Tx, PoolL2tx and L2Coordinator txn later on. Please for now, review this one. Most of the removed things were already commented out, it will not take more than 20 min. Also this is a low  priority task so I dont want to spend much time for both of us on it.

Shailu-s commented:
> > This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4 https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4
> > I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.
> 
> Agreed! It'd be great to break this PR down into multiple PRs. Separate PRs for L2Db, purger, l2tx, etc. would make it easier for reviewing and rolling back later on if something goes wrong 🙏

It will be time taking task to undo changes from this PR and create a new one for functions which I have already removed. I will create separate PRs for L2Tx, PoolL2tx and L2Coordinator txn later on. Please for now, review this one. Most of the removed things were already commented out, it will not take more than 20 min. Also this is a low  priority task so I dont want to spend much time for both of us on it.

Shailu-s committed:
Merge branch 'develop' into fix/remove-l2db

Intizar-T commented on sequencer/tokamak-sybil-resistance:
I wonder what this 33.3MB binary file is 🤔

Shailu-s replied:
Ohh thats build file generated by go while testing compilation. Thanks for pointing out, I will remove this from here.

Shailu-s replied:
Done

Intizar-T commented:
> > > This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4 https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4
> > > I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.
> > 
> > 
> > Agreed! It'd be great to break this PR down into multiple PRs. Separate PRs for L2Db, purger, l2tx, etc. would make it easier for reviewing and rolling back later on if something goes wrong 🙏
> 
> It will be time taking task to undo changes from this PR and create a new one for functions which I have already removed. I will create separate PRs for L2Tx, PoolL2tx and L2Coordinator txn later on. Please for now, review this one. Most of the removed things were already commented out, it will not take more than 20 min. Also this is a low priority task so I dont want to spend much time for both of us on it.

okay, then how about cleaning up the migrations files in this PR

Shailu-s commented:
> > > > This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4 https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4
> > > > I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.
> > > 
> > > 
> > > Agreed! It'd be great to break this PR down into multiple PRs. Separate PRs for L2Db, purger, l2tx, etc. would make it easier for reviewing and rolling back later on if something goes wrong 🙏
> > 
> > 
> > It will be time taking task to undo changes from this PR and create a new one for functions which I have already removed. I will create separate PRs for L2Tx, PoolL2tx and L2Coordinator txn later on. Please for now, review this one. Most of the removed things were already commented out, it will not take more than 20 min. Also this is a low priority task so I dont want to spend much time for both of us on it.
> 
> okay, then how about cleaning up the migrations files in this PR

Sure, Do you mean I should include clean up of migration files in this PR ?

Intizar-T commented:
> > > > > This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4 https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4
> > > > > I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.
> > > > 
> > > > 
> > > > Agreed! It'd be great to break this PR down into multiple PRs. Separate PRs for L2Db, purger, l2tx, etc. would make it easier for reviewing and rolling back later on if something goes wrong 🙏
> > > 
> > > 
> > > It will be time taking task to undo changes from this PR and create a new one for functions which I have already removed. I will create separate PRs for L2Tx, PoolL2tx and L2Coordinator txn later on. Please for now, review this one. Most of the removed things were already commented out, it will not take more than 20 min. Also this is a low priority task so I dont want to spend much time for both of us on it.
> > 
> > 
> > okay, then how about cleaning up the migrations files in this PR
> 
> Sure, Do you mean I should include clean up of migration files in this PR ?

yes

Shailu-s committed:
removed basic l2 specs from migrations

Shailu-s committed:
Merge branch 'develop' into fix/remove-l2db

Shailu-s commented:
> > > > > > This PR contains these two tasks: https://www.notion.so/tokamak/Remove-Purger-164d96a400a38022aaacebcd45b9c9eb?pvs=4 https://www.notion.so/tokamak/Remove-L2Db-related-code-14bd96a400a380d184dada0f81ff5d0e?pvs=4
> > > > > > I have mostly worked on to remove L2DB, purger, some instances of L2tx and coordinator transaction from multiple files. I will create two more tickets to remove L2Tx, PoolL2 txns and L1Coordinator txns. This PR has already more than 800 changes so it will be good to divide this into multiple task and PRs. Let me your suggestions.
> > > > > 
> > > > > 
> > > > > Agreed! It'd be great to break this PR down into multiple PRs. Separate PRs for L2Db, purger, l2tx, etc. would make it easier for reviewing and rolling back later on if something goes wrong 🙏
> > > > 
> > > > 
> > > > It will be time taking task to undo changes from this PR and create a new one for functions which I have already removed. I will create separate PRs for L2Tx, PoolL2tx and L2Coordinator txn later on. Please for now, review this one. Most of the removed things were already commented out, it will not take more than 20 min. Also this is a low priority task so I dont want to spend much time for both of us on it.
> > > 
> > > 
> > > okay, then how about cleaning up the migrations files in this PR
> > 
> > 
> > Sure, Do you mean I should include clean up of migration files in this PR ?
> 
> yes

Removed basic L2 specs from migration files, but I notes there are still many things to remove from migration like auction, wdelayer, pool specs which are needed etc. Currently we have high priority task to test sync so I will create tasks for this migration removal.

Shailu-s committed:
fixed issue after merging

Intizar-T reviewed (approved):
lgtm!
